catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -24,16 +24,21 @@ This is _not_ the date when the item was first published on the source site.
 
 ## Description
 
-The timestamp of when the media item was last updated in Openverse catalog.
+The timestamp of the last time any change was made to the media item. Unlike
+`last_synced_with_source`, this can also be a change from a data cleaning step,
+e.g. updating license URL in the `meta_data`, or fixing the URL using the
+`batched_update` DAG.
```

Comment:
Sorry for being ambiguous: this is _not the current behavior_, but I think it probably should be IMO!

Reply:
Should we open an issue to make `batched_update` DAG update the `updated_on`, @AetherUnbound ? This will make it the _current behavior_ since we decided to try to move most updates to the `batched_update`.

Reply:
That's a great idea!